src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -620,6 +646,399 @@ components:
         - STEP_STARTED
         - STEP_COMPLETED
 
+    CreateCiagInductionRequest:
+      title: CreateCiagInductionRequest
+      type: object
+      description: A request to create a CIAG Induction. Migrated from hmpps-ciag-careers-induction-api.
+      properties:
+        requestDTO:
+          $ref: '#/components/schemas/CreateCiagInductionRequestData'
+        oauth2User:
+          type: string
+
+    CreateCiagInductionRequestData:
+      title: CreateCiagInductionRequestData
+      description: The request data to create a CIAG Induction. Migrated from hmpps-ciag-careers-induction-api.
+      type: object
+      properties:
+        prisonId:
+          type: string
+          description: The ID of the Prison that that Prisoner is currently at.
+        prisonName:
+          type: string
+          description: The name of the Prison that that Prisoner is currently at.
+        hopingToGetWork:
+          $ref: '#/components/schemas/HopingToWork'
+        reasonToNotGetWorkOther:
+          type: string
+          description: The reason that is given when the Prisoner does not want to work. This is mandatory when 'reasonToNotGetWork' is set to 'OTHER'.
+        abilityToWorkOther:
+          type: string
+          description: A specific factor affecting the Prisoner's ability to work. This is mandatory when 'abilityToWork' is set to 'OTHER'.
+        abilityToWork:
+          uniqueItems: true
+          type: array
+          description: One or more factors affecting the Prisoner's ability to work.
+          items:
+            $ref: '#/components/schemas/AbilityToWorkFactor'
+        reasonToNotGetWork:
+          uniqueItems: true
+          type: array
+          description: One or more reasons the Prisoner has given not to get work after leaving Prison.
+          items:
+            $ref: '#/components/schemas/ReasonNotToWork'
+        workExperience:
+          $ref: '#/components/schemas/PreviousWorkRequest'
+        skillsAndInterests:
+          $ref: '#/components/schemas/SkillsAndInterestsRequest'
+        qualificationsAndTraining:
+          $ref: '#/components/schemas/EducationAndQualificationsRequest'
+        inPrisonInterests:
+          $ref: '#/components/schemas/PrisonWorkAndEducationRequest'
+      required:
+        - desireToWork
```

Comment:
Did you remove this field from the request body? Its marked as required here, but I can't see it?

Reply:
Good spot, I'll remove it here too 👍